### ISSUE TITLE
Version 1.5 forces redux-thunk to be <2.0

### ISSUE THREAD

**Issue by Realtin** [dixieio/redux-json-api#58]
Hey,

in my project I'm using:
```   
"redux-json-api": "^1.4.1",
"redux-thunk": "^2.1.0"
```
so Travis installed redux-json-api 1.5 but since the peerDependency for `"redux-thunk": "^1.0.0" ` my build will fail because `"redux-thunk": "^2.1.0"` does not satisfy.

Is there a reason for forcing the dependency to be <2.0?
Maybe the peerDependency can be relaxed?

**Comment by Nopzen** [comment]
@Realtin - I think @egeriis did put on the `^1.0.0` due to thats where we are our own product currently, i will try to see if there are any severe changes from the `<2.0.0`, or having a talk with him, it would be lovely if possible to use the latest version of course :)

Let me have some investigation, and ill come back to you :)

**Comment by Nopzen** [comment]
@egeriis - I dont see any huge changes, to the `redux-thunk` upgrade other than if a user is running a `CommonJS`require.

The ES6 Import method no changes have been made, according to the notes from @gaearon is mentioning here [gaearon/redux-thunk](https://github.com/gaearon/redux-thunk)

Should we maybe update the peerDependency to `2.0.0` as minimum requirements ? or not enforce `1.0.0` as hard but rather set that version as minium ? 

My suggestion could be doing the either of the following:

```json
"peerDependency": {
    "redux-thunk": ">=1.0.0"
```
Or ...
```json
"peerDependency": {
    "redux-thunk": "^2.0.0"
```
since version two is the current stable version, it could have implications for users running a `CommonJS` setup, but we could put a note in the documentation regarding this, upgrade?

**Comment by egeriis** [comment]
There are no breaking changes for our environment https://github.com/gaearon/redux-thunk/releases/tag/v2.0.0.

I think the first solution would be fine.

**Comment by Nopzen** [comment]
Cool @egeriis will update that, 

@Realtin - Would this solution be satisfying for you, ill try to make a small release of this either today or tomorrow?

**Comment by Realtin** [comment]
yes that would be great @Nopzen :) Thank you!

**Comment by Nopzen** [comment]
@Realtin - This issue should now be resolved, and is released under the tag `1.5.1`

Thanks for raising this issue.
